.github/workflows/checks.yml

```diff
@@ -43,13 +43,11 @@ jobs:
         uses: ./.github/actions/setup
       - name: Install Foundry
         uses: foundry-rs/foundry-toolchain@v1
-      - name: Run Anvil Node
-        run: anvil --fork-url ${{ secrets.ETHEREUM_RPC_URL }} &
       - name: Test
         run: pnpm test:ci
         env:
-          ETHEREUM_RPC_URL: ${{ secrets.ETHEREUM_RPC_URL }}
```

Comment:
These RPCs will likely be used for example/debug scripts too so unless there's a strong reason can we keep them without the `VITE_` prefix?

Reply:
Fixed by using `import { loadEnv } from 'vite';` in `vitest.config.ts`